eng/pipelines/templates/steps/publish-cli.yml

```diff
@@ -78,6 +90,7 @@ steps:
       displayName: Copy install scripts to release (PR only)
 
   # Docker build
+  # TODO: Do we need to do a Docker Build in ARM64?
```

Comment:
I think long term what we want to do here is build a multi-arch image for both amd64 and arm64 and publish that as `azd:latest` or whatever.

https://www.docker.com/blog/multi-arch-build-and-images-the-simple-way/

Gives an overview of a "hard" and "simple" way to do this - but the "simple" way may be in practice harder for us and we might end up wanting to use the docker-manifest version since it might be easier to integrate in our pipelines vs getting a runner setup that can build everything at once.

I think it's also reasonable to defer this work as a separate work item.